index.html

```diff
@@ -101,100 +132,92 @@ <h3>Tools</h3>
-        <div class="work3 flex">
-          <!-- <div> -->
-          <img src="img/work_img/Portfolio2.png" alt="work1 image" id="work3img" />
-          <!-- </div> -->
-          <div id="work3text">
-            <h3 class="work-name">Multi-Post Stories</h3>
-            <p class="work-description">
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required. has been the industry's standard dummy text
-              ever since the 1500s, when an unknown printer took a standard
-              dummy text.
-            </p>
-            <ul class="work-skills grid-container">
-              <li class="gird">CSS</li>
-              <li class="gird">HTML</li>
-              <li class="gird">Bootstrap</li>
-              <li class="grid0">Ruby</li>
-            </ul>
-            <button>See Project</button>
-          </div>
-        </div>
-        <div class="work4 flex">
-          <!-- <div> -->
-          <img src="img/work_img/Portfolio3.png" alt="work1 image" id="work4img" />
-          <!-- </div> -->
-          <div id="work4text">
-            <h3 class="work-name">Multi-Post Stories</h3>
-            <p class="work-description">
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required. has been the industry's standard dummy text
-              ever since the 1500s, when an unknown printer took a standard
-              dummy text.
-            </p>
-            <ul class="work-skills grid-container">
-              <li class="gird">CSS</li>
-              <li class="gird">HTML</li>
-              <li class="gird">Bootstrap</li>
-              <li class="grid0">Ruby</li>
-            </ul>
-            <button>See Project</button>
-          </div>
-        </div>
+        <!-- insert the project html -->
+        <!-- <div class="work1 flex">
+                          <img src="img/work_img/Portfolio.png" alt="work1 image" id="work1img" />
+                          <div id=work1text>
+                            <h3 class="work-name">Multi-Post Stories</h3>
+                            <p class="work-description">
+                              A daily selection of privately personalized reads; no accounts or
+                              sign-ups required. has been the industry's standard dummy text
+                              ever since the 1500s, when an unknown printer took a standard
+                              dummy text.
+                            </p>
+                            <ul class="work-skills grid-container">
+                              <li class="gird">CSS</li>
+                              <li class="gird">HTML</li>
+                              <li class="gird">Bootstrap</li>
+                              <li class="grid0">Ruby</li>
+                            </ul>
+                            <button class="btn-window" id="work1">See Project</button>
+                          </div>
```

Comment:
- [ ] Clean up this file. It is a better practice to remove pieces of commented-out code from you repo.